addons/account_payment_invoice_online_payment_patch/views/account_portal_templates.xml

```diff
@@ -23,6 +23,12 @@
                     <t t-if="invoice.state == 'posted' and invoice.payment_state in ('paid', 'in_payment') or last_tx.state == 'done'">
                         <span class="badge rounded-pill text-bg-success"><i class="fa fa-fw fa-check"></i><span class="d-none d-md-inline"> Paid</span></span>
                     </t>
+                    <t t-if="invoice.state == 'posted' and invoice.payment_state == 'reversed'">
+                        <span class="badge rounded-pill text-bg-success"><i class="fa fa-fw fa-check"></i><span class="d-none d-md-inline"> Reversed</span></span>
+                    </t>
+                    <t t-if="invoice.state == 'cancel'">
+                        <span class="badge rounded-pill text-bg-danger"><i class="fa fa-fw fa-remove"></i><span class="d-none d-md-inline"> Cancelled</span></span>
```

Comment:
```suggestion
                        <span class="badge rounded-pill text-bg-danger"><i class="fa fa-fw fa-remove"></i><span class="d-none d-md-inline"> Canceled</span></span>
```

Reply:
Same problem some lines below, and wanna keep c/p with existing xml code.